controller/model/_delete.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+import (
+	"github.com/openziti/foundation/transport/tcp"
+	"google.golang.org/appengine/socket"
+	"net/http"
+)
+
+func hi() {
```

Comment:
this might be my favorite function if this is intentional. does `hi` fulfill some interface? maybe a comment if this is useful in some way i'm not seeing?

Reply:
Hahah I'll remove it.